syntax = "proto3";
package agent;

message FlowInfo{
    fixed32 src_ip = 1;
    fixed32 dest_ip = 2;
    int32 packets_sent = 3;
    int32 retransmissions = 4;
    string flow_id = 5;
}

message Flows{
    repeated FlowInfo flow_info = 1;
}
